addons/test_mail/tests/common.py

```diff
@@ -27,6 +27,7 @@ def setUpClass(cls):
         super(BaseFunctionalTest, cls).setUpClass()
 
         cls.user_employee = mail_new_test_user(cls.env, login='ernest', groups='base.group_user', signature='--\nErnest', name='Ernest Employee')
+        cls.user_employee_2 = mail_new_test_user(cls.env, login='alexander', groups='base.group_user', signature='--\nAlexander', name='Alexander Employee')
```

Comment:
Side note: I think it is better to define this employee only in the test using it to reduce the test footprint.

Moreover it could be interesting to test that a user not having access is not added as follower.